### PR TITLE
Completed javascript-operators-and-expressions exercise. Bryan Valdovinos

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
Screen Shot 2022-05-16 at 1 00 36 PM

### CODE REVIEW

**Comment by bryanvaldovinos** [comment]
Screen Shot 2022-05-16 at 3 01 04 PM

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+var width;
+var height;
+var area;
+
+width = 5;
+height = 6;
+area = width * height;
```
Comment (codingcodymiller) [inline javascript-operators-and-expressions/main.js:7]: This is not necessary in JavaScript, you can assign a value on the same line you declare a variable 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (bryanvaldovinos) [reply javascript-operators-and-expressions/main.js]: just fixed, thanks for the heads up!

Comment (bryanvaldovinos) [reply javascript-operators-and-expressions/main.js]: Screen Shot 2022-05-16 at 3 35 56 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**